plugins/module_utils/pfsense.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, module, config='/cf/conf/config.xml'):
         self.root = self.tree.getroot()
         self.config_version = float(self.get_element('version').text)
         self.aliases = self.get_element('aliases', create_node=True)
+        self.dhcpd = self.get_element('dhcpd', create_node=True)
```

Comment:
I don't think we should always be creating a dhcpd element.  Drop this and add `create_node=True` to the module init get_element.